haddock-api/src/Haddock/GhcUtils.hs

```diff
@@ -449,7 +450,7 @@ modifySessionDynFlags f = do
 -- | A variant of 'gbracket' where the return value from the first computation
 -- is not required.
 gbracket_ :: ExceptionMonad m => m a -> m b -> m c -> m c
-gbracket_ before_ after thing = gbracket before_ (const after) (const thing)
+gbracket_ = MC.bracket_
```

Comment:
`gbracket_` was defined just to work around the annoying type of `gbracket`. Now that we have `MC.bracket_` with the right type, I think it would be better to just replace all occurences of `gbracket_` with `MC.bracket_`. Even better, I think there is only once such occurrence.

https://github.com/haskell/haddock/blob/7c34f89d801bafba3162bc091dd486cbb9e9fd5b/haddock-api/src/Haddock/Utils.hs#L407